airbyte-integrations/connectors/source-postgres/src/main/java/io/airbyte/integrations/source/postgres/PostgresSource.java

```diff
@@ -399,6 +406,11 @@ public List<AutoCloseableIterator<AirbyteMessage>> getIncrementalIterators(final
                                                                              final Instant emittedAt) {
     final JsonNode sourceConfig = database.getSourceConfig();
     if (PostgresUtils.isCdc(sourceConfig) && shouldUseCDC(catalog)) {
+      if (sourceConfig.has("cdc_via_ctid") && sourceConfig.get("cdc_via_ctid").asBoolean()) {
```

Comment:
We should add similar flag for user cursor so it can be in a controlled roll out after merge to master 
@nguyenaiden, @prateekmukhedkar

Reply:
Can we create a seperate file with these config flags defined as constants?

Reply:
I can add this flag in a separate file in my PR.